openshift/4.0/templates/base-images/dotnet50/Dockerfile.ocp3

```diff
@@ -0,0 +1,12 @@
+FROM registry.redhat.io/ubi8/dotnet-50
```

Comment:
Attempts to build dotnet 5.0 with Red Hat images.  So far not successful, but I'd like to solve this eventually.  This file is for OCP3.

Reply:
if this is for ocp3 it seems weird to have this in the openshift/4.0 directory.

Reply:
It's technically in this folder because it required building the base image in OCP3 and then manually copying to OCP4...